common/local-settings.ts

```diff
@@ -16,6 +16,7 @@ export interface ShieldBatteryAppSettingsData {
 }
 
 export interface LocalSettingsData extends ShieldBatteryAppSettingsData {
+  openAppAtSystemStart: boolean
```

Comment:
Sorry to do this to you, but maybe `runAppAtSystemStart` might be a bit more inline with our new label text :d. Ignore if you don't feel like changing it anymore.